### DISCUSSION TITLE
Anyone else having issue of updating packages on RHEL box where Stig Manager Docker Containers live?

### DISCUSSION THREAD

**Discussion by st3v1s** [NUWCDIVNPT/stig-manager#1092] [General]
Looking for work-arounds or resolution to a problem I'm seeing.  Running a RHEL8 virtual machine that has Docker installed for the containers (keycloak, stig-manager, nginx, mysql) to run Stig Manager with CAC authentication.

When I try to run yum update, the containers fail to start up.  I have only been successful updating Linux packages by excluding the following kernels:  kernel, docker, container, selinux-container.

Anyone else face this before and have a solution?

I'm currently at Linux kernel 4.18.0-425.3.1.el8.x86_64 and Docker version 20.10.21

Tried googling which suggested recreating overlay networks but not sure.

Thanks!

**Comment by Rcline45** [comment]
Are you able to post any logs for the containers or errors during compose up? I'm running RHEL8/podman haven't had any issues with yum update.

**Reply by st3v1s** [reply]
I will try and gather logs after hours as this is a production system.

That was another question I had....I'm running RHEL8, any reason I would need to migrate my Docker build to podman?  I know support for Docker ended in RHEL7 but I should be good using either on RHEL8.

**Reply by st3v1s** [reply]
So it looks like stigmanager is having connection issues to mysql and then fails....error says component: mysql, type: preflight, data, success: false, message, EHOSTUNREACH:  IPADDRESS:50001
Last line shows:  stigmanager, shutdown, failed to setup dependencies

**Reply by cd-rite** [reply]
You might have it configured that way deliberately, but it looks like the STIGMan API is trying to reach MySQL on port 50001.... Usually MySQL uses port 3306 (Assuming "IPADDRESS" is just a placeholder for the actual IP).
